lib/rdl/constraint.rb

```diff
@@ -1,4 +1,5 @@
 require 'csv'
+require 'method_source'
```

Comment:
@jeffrey-s-foster When it would go to write out the CSV file, it would crash when it tried to call the `comment` method on a method object. It's a little but unnerving, because I'm not sure why it was working before---there are two spots in RDL where this require gets called inside of some methods, so I'm not really sure what the code path was to get there.

Still, adding it here fixed the crash.